### ISSUE TITLE
if policyString is emtpy crashes!

### ISSUE THREAD

**Issue by designfrontier** [ansble/monument#206]
:cry:

**Comment by designfrontier** [comment]
Will need to patch 2.1.x, 2.2.x, 2.3.x

**Comment by designfrontier** [comment]
The root cause of this (which will be detailed in the release notes and the readme is that it expects an array of directives
```
'styleSrc': [`'self'`, 'cdn.google.com']
```

and when it instead gets a string:
```
'styleSrc': `'self' cdn.google.com`
```

it blows up for specific versions of Firefox.

**Comment by designfrontier** [comment]
closed by 2.3.3, 2.2.3, 2.1.2

**Comment by designfrontier** [comment]
See #207 #208 #209 #210
